src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -358,6 +399,12 @@ void DynamoViewModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
                     // When workspace is changed(e.g. from home to custom), close InCanvasSearch.
                     OnRequestShowInCanvasSearch(ShowHideFlags.Hide);
                     break;
+                case "NodeRunChecked":
```

Comment:
See my comments elsewhere about this menu item. If we need this, I don't think it should be checkable, it should just be an action like Copy or Paste, which bulk freezes a bunch of nodes. Is that the intention?